<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MenuPrincipale</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.png</normaloff>icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
	background-color:#eeeeee;
}
QMenu::item{
	background-color: white;
}
QMenu::item:selected{
	background-color: qlineargradient(spread:pad, x1:0.548, y1:1, x2:0.469455, y2:0, stop:0.627119 rgba(242, 101, 101, 255), stop:1 rgba(255, 146, 117, 255))
}
QMenuBar::item{
	background-color: white;
}
QMenuBar::item:selected{
	
	background-color: qlineargradient(spread:pad, x1:0.548, y1:1, x2:0.469455, y2:0, stop:0.627119 rgba(242, 101, 101, 255), stop:1 rgba(255, 146, 117, 255));
}</string>
  </property>
  <property name="iconSize">
   <size>
    <width>200</width>
    <height>200</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="img">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>icon.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>690</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menugeneral">
    <property name="title">
     <string>Informations génerales</string>
    </property>
    <widget class="QMenu" name="menubibiotheque">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Bibliotheque</string>
     </property>
     <property name="icon">
      <iconset theme="icon.png">
       <normaloff>:/icon/icon.png</normaloff>:/icon/icon.png</iconset>
     </property>
     <addaction name="actionemploye"/>
     <addaction name="actionservice"/>
     <addaction name="actionstr"/>
     <addaction name="actionprojet"/>
     <addaction name="actionetat"/>
    </widget>
    <addaction name="menubibiotheque"/>
    <addaction name="actionUtilisateurs"/>
    <addaction name="separator"/>
    <addaction name="actionparametre"/>
    <addaction name="actiondeconnection"/>
    <addaction name="actionquitter"/>
   </widget>
   <widget class="QMenu" name="menuliste_des_materiels">
    <property name="title">
     <string>Liste des materiels</string>
    </property>
    <addaction name="actionmateriel_informatique"/>
    <addaction name="actionmateriel_MGX"/>
   </widget>
   <widget class="QMenu" name="menutransfer">
    <property name="title">
     <string>Transfers</string>
    </property>
    <addaction name="actionListe_des_transfers"/>
   </widget>
   <widget class="QMenu" name="menufournisseur">
    <property name="title">
     <string>Fournisseurs</string>
    </property>
    <addaction name="actioncarnet_d_adresses"/>
   </widget>
   <widget class="QMenu" name="menuoutil">
    <property name="title">
     <string>Outils</string>
    </property>
    <addaction name="actioncalculatrice"/>
   </widget>
   <widget class="QMenu" name="menuA_propos">
    <property name="title">
     <string>A propos</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>?</string>
    </property>
   </widget>
   <addaction name="menugeneral"/>
   <addaction name="menuliste_des_materiels"/>
   <addaction name="menutransfer"/>
   <addaction name="menufournisseur"/>
   <addaction name="menuoutil"/>
   <addaction name="menuA_propos"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionparametre">
   <property name="text">
    <string>Paramètres</string>
   </property>
  </action>
  <action name="actiondeconnection">
   <property name="text">
    <string>Déconnexion</string>
   </property>
  </action>
  <action name="actionquitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionemploye">
   <property name="text">
    <string>Employés</string>
   </property>
  </action>
  <action name="actionstr">
   <property name="text">
    <string>Structure</string>
   </property>
  </action>
  <action name="actionprojet">
   <property name="text">
    <string>Projet</string>
   </property>
  </action>
  <action name="actionservice">
   <property name="text">
    <string>Service</string>
   </property>
  </action>
  <action name="actionmateriel_informatique">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Materiels informatiques</string>
   </property>
  </action>
  <action name="actionmateriel_MGX">
   <property name="text">
    <string>Materiels généraux</string>
   </property>
  </action>
  <action name="actiontransfer_ext">
   <property name="text">
    <string>transfer ext</string>
   </property>
  </action>
  <action name="actionaffectation_int">
   <property name="text">
    <string>affectation int</string>
   </property>
  </action>
  <action name="actioncomptage_gen">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Comptage général</string>
   </property>
  </action>
  <action name="actioninvestissement">
   <property name="text">
    <string>investissement </string>
   </property>
  </action>
  <action name="actioninv_faible_v">
   <property name="text">
    <string>inv faible v</string>
   </property>
  </action>
  <action name="actioninv">
   <property name="text">
    <string>inv</string>
   </property>
  </action>
  <action name="actioncarnet_d_adresses">
   <property name="text">
    <string>Carnet d'adresses</string>
   </property>
  </action>
  <action name="actioncalculatrice">
   <property name="text">
    <string>Calculatrice</string>
   </property>
  </action>
  <action name="actionetat">
   <property name="text">
    <string>Codification</string>
   </property>
  </action>
  <action name="actionListe_des_utilisateurs">
   <property name="text">
    <string>Liste des utilisateurs</string>
   </property>
  </action>
  <action name="actionChanger_Mot_de_passe">
   <property name="text">
    <string>Changer Mot de passe</string>
   </property>
  </action>
  <action name="actionComptage_informatique">
   <property name="text">
    <string>Comptage informatique</string>
   </property>
  </action>
  <action name="actionComptage_moyennes_g_n_rales">
   <property name="text">
    <string>Comptage moyennes générales</string>
   </property>
  </action>
  <action name="actionGestion_des_transfers">
   <property name="text">
    <string>Liste des transfers</string>
   </property>
  </action>
  <action name="actionListe_des_transfers">
   <property name="text">
    <string>Liste des transfers</string>
   </property>
  </action>
  <action name="actionUtilisateurs">
   <property name="text">
    <string>Utilisateurs</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionquitter</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
